rpms/s3/s3rpm.spec

```diff
@@ -126,21 +136,40 @@ S3 server provides S3 REST API interface support for Motr object storage.
 %prep
 %setup -n %{name}-%{version}-%{_s3_git_ver}
 
+################################
+# pre install/upgrade section
+################################
 %pre
-if [ -f /opt/seagate/cortx/s3/conf/s3config.yaml.sample ]; then
-cp -f /opt/seagate/cortx/s3/conf/s3config.yaml.sample /tmp/s3config.yaml.sample.old
-fi
-if [ -f /opt/seagate/cortx/s3/s3backgrounddelete/config.yaml.sample ]; then
-    cp -f /opt/seagate/cortx/s3/s3backgrounddelete/config.yaml.sample /tmp/config.yaml.sample.old
-fi
-if [ -f /opt/seagate/cortx/auth/resources/keystore.properties.sample ]; then
-    cp -f /opt/seagate/cortx/auth/resources/keystore.properties.sample /tmp/keystore.properties.sample.old
-fi
-if [ -f /opt/seagate/cortx/auth/resources/authserver.properties.sample ]; then
-    cp -f /opt/seagate/cortx/auth/resources/authserver.properties.sample /tmp/authserver.properties.sample.old
+if [ $1 == 1 ];then
+    echo "S3 RPM Pre Install section started"
+    echo "S3 RPM Pre Install section completed"
+elif [ $1 == 2 ];then
+    echo "S3 RPM Pre Upgrade section started"
+    echo "Backing up .sample config file to .old"
+    if [ -f /opt/seagate/cortx/s3/conf/s3config.yaml.sample ]; then
+        cp -f /opt/seagate/cortx/s3/conf/s3config.yaml.sample /tmp/s3config.yaml.sample.old
+    fi
+    if [ -f /opt/seagate/cortx/s3/s3backgrounddelete/config.yaml.sample ]; then
+        cp -f /opt/seagate/cortx/s3/s3backgrounddelete/config.yaml.sample /tmp/config.yaml.sample.old
+    fi
+    if [ -f /opt/seagate/cortx/s3/s3backgrounddelete/s3_cluster.yaml.sample ]; then
+        cp -f /opt/seagate/cortx/s3/s3backgrounddelete/s3_cluster.yaml.sample /tmp/s3_cluster.yaml.sample.old
+    fi
+    if [ -f /opt/seagate/cortx/auth/resources/keystore.properties.sample ]; then
+        cp -f /opt/seagate/cortx/auth/resources/keystore.properties.sample /tmp/keystore.properties.sample.old
+    fi
+    if [ -f /opt/seagate/cortx/auth/resources/authserver.properties.sample ]; then
+        cp -f /opt/seagate/cortx/auth/resources/authserver.properties.sample /tmp/authserver.properties.sample.old
+    fi
+    echo "S3 RPM Pre Upgrade section completed"
 fi
 
+################################
+# build section
+################################
 %build
+echo "S3 RPM Build section started"
```

Comment:
Add uniform prefix to all echo statements in 
1)RPM spec
2)Merge.sh file

Reply:
added [cortx-s3server-rpm] as prefix